Windows/build_package.cmd

```diff
@@ -170,6 +170,11 @@ xcopy /y include\openssl\* %SGXSSL_ROOT%\package\include\openssl\
 xcopy /y include\crypto\* %SGXSSL_ROOT%\package\include\crypto\
 
 echo "Done building OpenSSL for %my_Platform% | %my_Configuration%. Building Intel® Software Guard Extensions SSL libraries  %date% %time%"
+
+if "%TEST_MODE%" equ "NOTEST" (
+goto end
```

Comment:
If the condition is true, the build steps for sgxssl libs will also be skipped, right?

Reply:
Yes. The purpose of this PR is to build the crypto library only, when sgx_edger8r is not available in some particular cases.  And when sgx_edger8r is not available, it will not be able to build trusted and untrusted libraries.

Reply:
Then you need also to make Linux version consistent with the purpose. Currently Linux only skips test build and still build sgxssl libs. 
And it is not proper to use the macro NO_BUILDTEST in this case.

Reply:
Linux script has been updated. And also change the macro name in the Windows script. The commands have been changed to:
`CRYPTO_ONLY=1 make`
and
`build_package.cmd Platform_Configuration OpenSSL_Version no-clean CRYPTO_ONLY`